### ISSUE TITLE
Hebrew Translation

### ISSUE THREAD

**Issue by moriel5** [ThirtyDegreesRay/OpenHub#5]
I have (sort of) finished translating OpenHub to hebrew (using Stringlate really helped me organize things towards the end).
The possible oddity in the title is due to Stringlate forcing me to put %x in the title (I have no idea what will show up instead yet)

Issues:
1. There a few lines (such as Theme Accent Color) that I may change after the next release, as I am not sure about the quality of my translation of those lines.

2. I foresee issues with the lines "days ago", "hours ago", "minutes ago", and "seconds ago", since in hebrew the context is different (i.e. "5 minutes ago" becomes something like "before 5 minutes").

Update: I had fixed the oddities from the browser, "strings.xml" incoming.
Update2: Here is the file, I had to archive it as GitHub would not allow me to upload .xml files.

[he-IL.zip](https://github.com/ThirtyDegreesRay/OpenHub/files/1343923/he-IL.zip)

**Comment by ThirtyDegreesRay** [comment]
Thanks, I have add Hebrew to OpenHub, but something strange, everything show as right-to-left from original left-to-right, is this normal?

Something wrong with uploading screenshots, I can't show you the screenshots. but I will release v1.2.1 tonight in Beijing time, you can download and experience it.

About your issues:

1.Okay, next time you can pull request to OpenHub, it's more convenient.

2.Thses line we can prefert later.

In a word, you experience v1.2.1 first, then we can perfect it if something was improper.

**Comment by ThirtyDegreesRay** [comment]
@moriel5 OpenHub [v1.2.1](https://github.com/ThirtyDegreesRay/OpenHub/releases/tag/v1.2.1) is released, enjoy it.

**Comment by moriel5** [comment]
Thanks, I'll test it now.
And about RTL, it's fine, hebrew is written from right to left.

**Comment by ThirtyDegreesRay** [comment]
Okay, editing previous comments and some other functions about issues will allow next version.

**Comment by moriel5** [comment]
Sorry about the lateness, I only recently came back from work (and this morning I had run out of battery).

That's great!

Now I only need two things to be able to completely move over to OpenHub: editing previous comments, and notifications (no need for system-wide notifications, though they would be nice, especially if they do not rely on Google's Play Services).

Regarding OpenHub, I can only wish you the best (and I'll translate to hebrew when I could).

By the way, when I had tried to post this from OpenHub, I only got an error that said "Page not found", could this be what you were referring to when you wrote about fixes in relation to issues?
